pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -211,32 +211,21 @@ func (s *Service) handle(msg *message) error {
 		// TODO pass invitation id #397
 		s.sendMsgEvents(&service.StateMsg{
 			Type: service.PostState, Msg: msg.Msg, StateID: next.Name(),
-			Properties: s.createEventProperties(msg.ThreadID, "")})
+			Properties: s.createEventProperties(msg.ThreadID, "", nil)})
 		logger.Infof("sent post event for state %s", next.Name())
 
 		next = followup
 	}
 	return nil
 }
 
-func (s *Service) createEventProperties(connectionID, invitationID string) *didExchangeEvent { //nolint: unparam
-	return &didExchangeEvent{connectionID: connectionID, invitationID: invitationID}
-}
-
-// didExchangeEvent implements didexchange.Event interface.
-type didExchangeEvent struct {
-	connectionID string
-	invitationID string
-}
-
-// ConnectionID returns DIDExchange connectionID.
-func (ex *didExchangeEvent) ConnectionID() string {
-	return ex.connectionID
-}
-
-// InvitationID returns DIDExchange invitationID.
-func (ex *didExchangeEvent) InvitationID() string {
-	return ex.invitationID
+func (s *Service) createEventProperties(connectionID,
```

Comment:
I'm on the fence on if the error version of this method should be split to its own method. I assume you had a rationale for this style?

Reply:
I think it makes sense to split this method. ll update the code.

Reply:
[Added](https://github.com/hyperledger/aries-framework-go/compare/c07ed53802da44eef1d1921b3f421ce1d620ab41..5562ad35e21a3e5cb071507e1e0854c0d9317794#diff-f761c91e753eba1c42a68076df06ce12L222).